rmw_implementation/src/functions.cpp

```diff
@@ -81,16 +85,18 @@ get_library()
 void *
 get_symbol(const char * symbol_name)
 {
-  Poco::SharedLibrary * lib = get_library();
+  std::shared_ptr<rcpputils::SharedLibrary> lib = get_library();
+
   if (!lib) {
-    // error message set by get_library()
```

Comment:
Why was this comment removed? It seems the function still sets an error message.